DB/src/main/java/io/deephaven/db/v2/SourceTableMap.java

```diff
@@ -73,7 +73,7 @@ public SourceTableMap(@NotNull final TableDefinition tableDefinition,
             subscriptionBuffer = new TableLocationSubscriptionBuffer(tableLocationProvider);
             pendingLocationStates = new IntrusiveDoublyLinkedQueue<>(IntrusiveDoublyLinkedNode.Adapter.<PendingLocationState>getInstance());
             readyLocationStates = new IntrusiveDoublyLinkedQueue<>(IntrusiveDoublyLinkedNode.Adapter.<PendingLocationState>getInstance());
-            processNewLocationsLiveTable = new InstrumentedLiveTable(SourceTableMap.class.getSimpleName() + '[' + tableDefinition.getNamespace() + ',' + tableDefinition.getName() + ']' + "-processPendingLocations") {
+            processNewLocationsLiveTable = new InstrumentedLiveTable(SourceTableMap.class.getSimpleName() + "-processPendingLocations") {
```

Comment:
A better choice would be to replace the removed code fragment with:
`'[' + tableLocationProvider + ']' +`